### PR TITLE
Add root breadcrumb title to add page form

### PR DESCRIPTION
**Pull request by phalkunz**
Issue: https://github.com/silverstripe/silverstripe-cms/issues/2055

### CODE REVIEW

File: lang/en.yml [hunk 1]
```diff
@@ -72,6 +72,7 @@ en:
     ParentMode_child: 'Under another page'
     ParentMode_top: 'Top level'
     Title: 'Add page'
+    ROOT_BREADCRUMB_TITLE: 'Pages'
```
Comment (tractorcow) [inline lang/en.yml]: Duplicate of `SilverStripe\CMS\Controllers\CMSPagesController.MENUTITLE`. Please use the existing key instead.

Comment (flamerohr) [reply lang/en.yml]: Update made

**Comment by tractorcow** [comment]
It's not clickable, but it still is an improvement. :)

**Comment by tractorcow** [comment]
Can be merged on green.

**Comment by dhensby** [comment]
linting issues

**Comment by tractorcow** [comment]
Fixed linting issues on 4.0 branch and rebased this on top.

**Pull request merged**